Software/Development/SemesterProject/System/systemInit.c

```diff
@@ -137,13 +137,13 @@ void systemInit(void)
     // The  RamfuncsLoadStart, RamfuncsLoadSize, and RamfuncsRunStart
     // symbols are created by the linker. Refer to the F2808.cmd file.
     //
-    //memcpy(&RamfuncsRunStart, &RamfuncsLoadStart, (Uint32)&RamfuncsLoadSize);
+    memcpy(&RamfuncsRunStart, &RamfuncsLoadStart, (Uint32)&RamfuncsLoadSize);
 
     //
     // Call Flash Initialization to setup flash waitstates
     // This function must reside in RAM
     //
-    //InitFlash();
+    InitFlash();
 
```

Comment:
This changes are already part of another pull request¡!¡!

BTW do you like the new symbols?¿!¡

Reply:
I really do ¡¡¡¡

Reply:
But as the change is the same it shouldn't give a problem

Reply:
OKAY¡!¡!